File: src/posidon/potassium/universe/WorldSave.java
package posidon.potassium.universe;

import posidon.potassium.backend.Player;

import java.io.Serializable;
import java.util.HashMap;

public class WorldSave implements Serializable {

    private static final long serialVersionUID = 1;
    public double time;

    // x, y, z, health, uranium
    public static HashMap<Integer, int[]> playerInfo = new HashMap<Integer, int[]>();

    public WorldSave() {
        this.time = Globals.getTime();
    }
}

File: src/posidon/potassium/universe/WorldSave.kt
package posidon.potassium.universe

import java.io.Serializable
import java.util.*

class WorldSave : Serializable {

    var time: Double = Globals.time

    companion object {
        private const val serialVersionUID: Long = 1
        // x, y, z, health, uranium
        var playerInfo = HashMap<Int, IntArray>()
    }
}